### PR TITLE
Openapi spec v2.7.2

### PR DESCRIPTION
**Pull request by sergiishamrai-okta**
## Description
Compatibility with Open API spec v2.7.2

### CODE REVIEW

File: integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthenticatorsIT.groovy [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import static org.hamcrest.Matchers.*
+
+/**
+ * Tests for {@code /api/v1/authenticators}.
+ * These tests requires the Org to be OIE enabled.
+ * @since 6.x.x
+ */
+class AuthenticatorsIT extends ITSupport {
+
+    @Test (groups = "bacon")
```
Comment (arvindkrishnakumar-okta) [inline integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthenticatorsIT.groovy:36]: Add a comment pointing to OKTA-433369 (note that it'll be enabled when that ticket is done).

File: integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthenticatorsIT.groovy [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    assertThat(authenticator, notNullValue())
+                    assertThat(authenticator.getName(), notNullValue())
+                    assertThat(authenticator.getStatus(), notNullValue())
+                    assertThat(authenticator.getKey(), notNullValue())
+            })
+    }
+
+    // temporarily disabled until we figure out if its safe to toggle authenticator status on an Org
+    // where several other management ITs run in parallel.
+    @Test (groups = "bacon", enabled = false)
```
Comment (arvindkrishnakumar-okta) [inline integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthenticatorsIT.groovy:59]: Same comment as https://github.com/okta/okta-sdk-java/pull/647/files#r719507600

File: integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy [hunk 3]
```diff
@@ -306,10 +306,13 @@ class UsersIT extends ITSupport implements CrudTestSupport {
         // 3. List roles for the user and verify added role
         assertPresent(user.listAssignedRoles(), role)
 
-        // 4. Remove role for the user
+        // 4. Verify added role
```
Comment (arvindkrishnakumar-okta) [inline integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy:309]: This has the potential to be a flaky test? Worth adding a sleep before this assertion?

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "@okta/openapi": "2.6.0"
+    "@okta/openapi": "2.7.1"
```
Comment (arvindkrishnakumar-okta) [inline package.json:3]: Can we wait for 2.7.2 and update it directly to that version?

Comment (sergiishamrai-okta) [reply package.json:3]: Sure, it should be 2.7.2
But for now, this version was not published yet so I am avoiding "not found error"

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @okta/openapi@2.7.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

Comment (sergiishamrai-okta) [reply package.json]: updated now

**Comment by sergiishamrai-okta** [comment]
Build failure fix: https://github.com/okta/okta-sdk-java/pull/650

**Review by arvindkrishnakumar-okta — APPROVED** [review]

**Pull request merged**